packages/bp/src/core/telemetry/stats-service.ts

```diff
@@ -34,6 +36,7 @@ export class StatsService {
     this.configStats.start()
     this.rolesStats.start()
     this.sdkStats.start()
+    this.userStats.start()
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
     this.refreshDB(DB_REFRESH_INTERVAL)
```

Comment:
Can you remove the comment and add void here please?

Reply:
Is this how we want to handle floating promises ? 

My personal pref would be to add a `void`, but always leave some sort of comment stating why the promise is floating. This will result on clearly expressing that the promise is intentionally floating.

What do you guys think?

Reply:
I also agree with @franklevasseur about using the comment to disable warning instead of using void. Intent is much clearer